mutornadomon/monitor.py

```diff
@@ -70,6 +107,12 @@ def __init__(self, host_limit=r'.*', request_filter=LOCALHOST, io_loop=None):
         self._GAUGES = {}
         self._reset_ephemeral()
 
+    def _publish(self):
+        try:
+            self.publisher(self.metrics)
```

Comment:
Actually, it makes sense to track the web application even if you're not exposing the HTTP endpoints. So I will factor all of this so that you can configure both publish vs. endpoint, and application vs. no application monitoring.